hello-microservices-core/src/main/java/matera/com/hellomicroservices/core/requests/CreatePersonRequest.java

```diff
@@ -0,0 +1,160 @@
+package matera.com.hellomicroservices.core.requests;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+
+@XmlRootElement
+@XmlAccessorType(XmlAccessType.FIELD)
+public class CreatePersonRequest {
```

Comment:
- Coding standard: always put an empty line between attributes and methods declaration
- Use tabspace = 4, seems that your code is using 8 (check how to configure your IDE)